src/AutoRest.CSharp.V3/Output/Builders/ClientBuilder.cs

```diff
@@ -362,19 +362,6 @@ PathSegment TextSegment(string text)
 
         private static int ToStatusCode(StatusCodes arg) => int.Parse(arg.ToString().Trim('_'));
 
-        private static RequestMethod? ToCoreRequestMethod(HttpMethod method) => method switch
-        {
-            HttpMethod.Delete => RequestMethod.Delete,
-            HttpMethod.Get => RequestMethod.Get,
-            HttpMethod.Head => RequestMethod.Head,
-            HttpMethod.Options => (RequestMethod?)null,
-            HttpMethod.Patch => RequestMethod.Patch,
-            HttpMethod.Post => RequestMethod.Post,
-            HttpMethod.Put => RequestMethod.Put,
-            HttpMethod.Trace => null,
-            _ => null
-        };
```

Comment:
I moved this to be in the same place were we have the 'opposite' method of it. It made more sense to keep them together so that if changes are needed, it is obvious that both need changed.